e2e/testcases/apiservice.bats

```diff
@@ -38,18 +56,24 @@ test_teardown() {
   wait::for -t 240 -- nomos::repo_synced
 }
 
-@test "importer and syncer resilient to bad APIService" {
+@test "importer and syncer resilient to flaky APIService" {
```

Comment:
Is the bats test still needed given we have an equivalent go test?

Reply:
So far it's still needed, the mono repo test on GKE clusters are still running bats tests and this is the quickest way to bring CI back to green. We are going to fully deprecate the bats tests in coming Fixit.